chia/wallet/nft_wallet/nft_wallet.py

```diff
@@ -471,6 +470,10 @@ async def update_metadata(
     def get_current_nfts(self) -> List[NFTCoinInfo]:
         return self.my_nft_coins
 
+    async def load_current_nft(self) -> List[NFTCoinInfo]:
+        self.my_nft_coins = await self.wallet_state_manager.nft_store.get_nft_list(wallet_id=self.wallet_id)
```

Comment:
do we store these both in RAM in `self.my_nft_coins` as well as in the `nft_store`?

Reply:
It was only in my_nft_coins as a part of the wallet metadata. We splitted the nft and wallet data later. I didn't touch this because of it can improve the performance a little bit. I'm fine to remove this in another PR.